index.ios.js

```diff
@@ -33,11 +35,17 @@ export default class Sketch extends React.Component {
       flex: 1,
       backgroundColor: 'transparent',
     };
+
+    this.state = {
+      imageData: props.imageData
```

Comment:
I found an issue where if the prop is `undefined` here, removing one `<Sketch` from the DOM and then rendering a new one later will cause the sketch to contain the old drawing. Adding `|| null` to this initial state causes the issue to go away--RCT will pass the null value inward as `nil`, but won't pass `undefined` as a `nil` to ObjC.

Reply:
- [x] done by fixing `defaultProps`